### ISSUE TITLE
Network graph filters get bypassed whenever you click on an empty part of the graph

### ISSUE THREAD

**Issue by darkbasic** [zwave-js/zwave-js-ui#3126]
### Checklist

- [ ]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

8.18.0

### ZwaveJS version

10.23.1

### Describe the bug

Whenever you set locations or node filters in the network graph they filter out all non matching nodes:

![image](https://github.com/zwave-js/zwave-js-ui/assets/1047358/e3ff13bb-6fb1-4163-a7f1-95c9fac75e63)

Unfortunately as soon as you click on an empty part of the network graph the filters get automatically bypassed and the whole graph gets displayed:

![image](https://github.com/zwave-js/zwave-js-ui/assets/1047358/97215f5c-bfc7-4291-8685-cfc88ea730e2)

### To Reproduce

Open zwavejs-ui in Home Assistant and go to the Network Graph, set a filter (location or node) and then click on an empty part of the graph.

### Expected behavior

Filters should always be respected unless you explicitly disable them.

### Additional context

The browser is Chromium and the screen resolution is 2560x1440, OS is Gentoo Linux ppc64le (Raptor CS Talos 2).

**Comment by robertsLando** [comment]
This is expected otherwise when a filter is set you have no way to switch back to overview, a change on filters will re-apply them

**Comment by darkbasic** [comment]
In my opinion the overview should be filtered when filters are enabled, it makes no sense to have it unfiltered: if I filter by location and I click on a node it shows just node, but when clicking on an empty part of the graph I expect to switch back to all the location nodes, not ALL the nodes in my network.
I suggest to make that change and add a switch to bypass filters, so you have a way to display all the nodes without having to remove the filters.

**Comment by duylong** [comment]
Same problem, I thought it was a bug, apparently not.

I filter by room, I click on a node, how do I go back without clicking on an empty area of ​​the graph? 

And I also agree, as long as the filter is selected, it should stay in that filter.

**Comment by darkbasic** [comment]
Yeah it's definitely a bug in my opinion and I even tried to get it fixed myself lately, but I couldn't find a way to connect my local development frontend to the remote zwave-js-ui backend because the HASSIO docker container doesn't expose the port to the host. I did quite a lot of work with vue3 + vuetify lately but unfortunately I noticed that the codebase is still vue2: hopefully it won't be too big of a difference.

**Comment by robertsLando** [comment]
@darkbasic the vuetify 3 switch is on my TODOs but requires lot of work, reason why I didn't do it already is just because vuetify was far from being so till few months ago

**Comment by darkbasic** [comment]
Yeah even now it's barely usable in production due to several missing key components. Most of them fortunately are available in Labs while others are easy enough to implement with some CSS hacks like the Material Sidesheet. That being said I love using latest Vuetify with vue3 and I think it's mature enough to begin transitioning.

**Comment by darkbasic** [comment]
If someone else is interested here is my PR to develop against an Ingress zwave-js-ui instance: https://github.com/zwave-js/zwave-js-ui/pull/3892
